src/com/reason/bs/BsCompilerImpl.java

```diff
@@ -53,11 +56,15 @@ public String getNamespace(@NotNull VirtualFile sourceFile) {
         return "";
     }
 
-    //region Compiler
-    @Nullable
     @Override
-    public VirtualFile findContentRoot(@NotNull Project project) {
-        return Platform.findORPackageJsonContentRoot(project);
+    @Deprecated
+    public Optional<VirtualFile> findFirstContentRoot(@NotNull Project project) {
+        return ORProjectManager.findFirstEsyContentRoot(project);
```

Comment:
a typo ? why esy ?

Reply:
That's a typo. Thanks for catching that.